charts/external-dns/tests/deployment-flags_test.yaml

```diff
@@ -71,6 +70,16 @@ tests:
           path: spec.template.spec.containers[?(@.name == "external-dns")].args
           content: "--txt-prefix=test-prefix"
```

Comment:
Would you please update the two tests above L50 and L23 accordingly ?

Reply:
I removed one test, and updated second